### PR TITLE
 update quadplane tuning initial values

### PR DESCRIPTION
**Pull request by Hwurzburg**

### CODE REVIEW

**Review by tridge — APPROVED** [review]

File: plane/source/docs/quadplane-vtol-tuning-process.rst [hunk 1]
```diff
@@ -100,15 +100,15 @@ vehicle. These parameters are critical to the tuning process.
 - :ref:`INS_GYRO_FILTER <INS_GYRO_FILTER>`: 80Hz for 5 inch props, 40Hz for 10 inch props, 20Hz for 20 inch props
 - :ref:`Q_A_ACCEL_P_MAX <Q_A_ACCEL_P_MAX>`: 110000 for 10 inch props, 50000 for 20 inch props, 20000 for 30 inch props
 - :ref:`Q_A_ACCEL_R_MAX <Q_A_ACCEL_R_MAX>`: 110000 for 10 inch props, 50000 for 20 inch props, 20000 for 30 inch props
-- :ref:`Q_A_ACCEL_Y_MAX <Q_A_ACCEL_Y_MAX>`: 27000 for 10 inch props, 18000 for 20 inch props, 9000 for 30 inch props
+- :ref:`Q_A_ACCEL_Y_MAX <Q_A_ACCEL_Y_MAX>`: 6000 for 6 inch props,27000 for 10 inch props, 18000 for 20 inch props, 9000 for 30 inch props
```
Comment (tridge) [inline plane/source/docs/quadplane-vtol-tuning-process.rst:103]: did you mean 60000 ?

File: plane/source/docs/quadplane-vtol-tuning-process.rst [hunk 2]
```diff
@@ -100,15 +100,15 @@ vehicle. These parameters are critical to the tuning process.
...
 - :ref:`Q_A_RAT_YAW_P <Q_A_RAT_YAW_P>`: 0.5 x :ref:`Q_A_ACCEL_Y_MAX <Q_A_ACCEL_Y_MAX>` / 4500
 
 
-- :ref:`Q_A_RAT_PIT_FLTD <Q_A_RAT_PIT_FLTD>`: :ref:`INS_GYRO_FILTER <INS_GYRO_FILTER>` / 2
+- :ref:`Q_A_RAT_PIT_FLTD <Q_A_RAT_PIT_FLTD>`: :ref:`INS_GYRO_FILTER <INS_GYRO_FILTER>` / 8
```
Comment (tridge) [inline plane/source/docs/quadplane-vtol-tuning-process.rst:107]: divided by 8 seems wrong for FLTD?

**Review by tridge — CHANGES_REQUESTED** [review]

File: plane/source/docs/quadplane-vtol-tuning-process.rst [hunk 3]
```diff
@@ -100,15 +100,15 @@ vehicle. These parameters are critical to the tuning process.
...
 - :ref:`Q_A_RAT_PIT_FLTT <Q_A_RAT_PIT_FLTT>`: :ref:`INS_GYRO_FILTER <INS_GYRO_FILTER>` / 2
 - :ref:`Q_A_RAT_RLL_FLTD <Q_A_RAT_RLL_FLTD>`: :ref:`INS_GYRO_FILTER <INS_GYRO_FILTER>` / 2
 - :ref:`Q_A_RAT_RLL_FLTT <Q_A_RAT_RLL_FLTT>`: :ref:`INS_GYRO_FILTER <INS_GYRO_FILTER>` / 2
-- :ref:`Q_A_RAT_YAW_FLTE <Q_A_RAT_YAW_FLTE>`: 2
+- :ref:`Q_A_RAT_YAW_FLTE <Q_A_RAT_YAW_FLTE>`: 0
```
Comment (IamPete1) [inline plane/source/docs/quadplane-vtol-tuning-process.rst:111]: I don't think we can just receomend turning it off, maybe it should say 2 for vehicles with fixed motors 0 for thrust vectored vehicles.

Comment (Hwurzburg) [reply plane/source/docs/quadplane-vtol-tuning-process.rst:111]: @lthall was pretty adamant about this being 0...as long as FTLD is set to a reasonable value

File: plane/source/docs/quadplane-vtol-tuning-process.rst (hunk 2 above)
Comment (Hwurzburg) [reply plane/source/docs/quadplane-vtol-tuning-process.rst:107]: think that was your suggestion when we talked about QuikTune....what do you and @lthall  suggest?

File: plane/source/docs/quadplane-vtol-tuning-process.rst (hunk 1 above)
Comment (Hwurzburg) [reply plane/source/docs/quadplane-vtol-tuning-process.rst:103]: no, both my and @andyp1per small vtols have this value, but whatever you guys think...I know I had one that did not stop swaying until it got lowered to this value although I did not experiment with intermediate values...just used Andy's value for his vtols

File: plane/source/docs/quadplane-vtol-tuning-process.rst (hunk 2 above)
Comment (tridge) [reply plane/source/docs/quadplane-vtol-tuning-process.rst:107]: for pitch and roll FLTD divided by 2 is a reasonable rule of thumb

File: plane/source/docs/quadplane-vtol-tuning-process.rst (hunk 3 above)
Comment (IamPete1) [reply plane/source/docs/quadplane-vtol-tuning-process.rst:111]: 0 for vectored vehicles. But these recommendations are for all vehicle types?

File: plane/source/docs/quadplane-vtol-tuning-process.rst (hunk 1 above)
Comment (Hwurzburg) [reply plane/source/docs/quadplane-vtol-tuning-process.rst:103]: changed to 1/4 multi values per Pete H///also updated curve labels

File: plane/source/docs/quadplane-vtol-tuning-process.rst (hunk 2 above)
Comment (Hwurzburg) [reply plane/source/docs/quadplane-vtol-tuning-process.rst:107]: reverted

File: plane/source/docs/quadplane-vtol-tuning-process.rst (hunk 3 above)
Comment (Hwurzburg) [reply plane/source/docs/quadplane-vtol-tuning-process.rst:111]: splitting out vectored yaw

**Review by IamPete1 — APPROVED** [review]
LGTM, thanks.

**Review by tridge — APPROVED** [review]

**Pull request merged**